### ISSUE TITLE
Did not find Log file

### ISSUE THREAD

**Issue by adailsonfilho** [mderriey/tfs-webhooks#1]
Hi there,

Firt of all thank you for mading this open source! I've runned the project in bebug and worked finne, but I could not found the log file, even with a full search for the file name in my computer. Later I modified to an absolute path, and it was there. But when I put it in IIS, no one of the options wrote the log file. I've already check for permissions, and all looks all right. Any ideias what could be the problem?

My best,

**Comment by mderriey** [comment]
Hi,

Thanks for your kind words, they're very appreciated.
I think I fixed this in #2.

A couple of things have changed:
 
 - The Serilog file sink didn't have the ability to flush to disk periodically. As this became available in a [newer version](https://github.com/serilog/serilog-sinks-file/pull/20), I updated the relevant NuGet packages and made the interval configurable in the `web.config`
 - I also found a way to execute code just when the app is shutting down, so a final flush should happen when you stop/recycle the application pool

Now I don't have IIS installed on my dev machine so I'm not sure about the second point. The code was executed when I clicked on the _Stop site_ menu entry in IIS Express, so hopefully it also works in IIS.

Cheers

**Comment by adailsonfilho** [comment]
Nice!!

Yeah! I've already updated these nugets. Actually I added [Serilog.Sinks.RollingFile](https://github.com/serilog/serilog-sinks-rollingfile) that handle the flushing. What explais it working locally with this change. My real problem was that the IIS default account (IIS_IUSRS) had not the necessary permissions, I said that they were OK, but I had missed the correct account. Thanks a lot for your time!

My best,
